style.css

```diff
@@ -0,0 +1,124 @@
+:root {
+  --d-green: #253d2c;
+  --l-green: #68ba7f;
+  --dark: #000000;
+  --white: #ffffff;
+  --grey: #d9d9d9;
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+i {
+  padding: 0.2rem;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+body {
```

Comment:
why are not there all the typography css. Here is the list:

- font-size
- color
- line-height
- font-weight
- font-family

**Add these values in css variables as well**